Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ValueListGroup/ValueListGroupItem.tsx

```diff
@@ -123,9 +123,9 @@ export default function ValueListGroupItems(props: IProps) {
             </div>
             <Warning
                 Message={'This will permanently delete this Value List Item and cannot be undone.'}
-                Show={showWarning} Title={'Delete ' + (record?.AltValue ?? record.Value)}
+                Show={showWarning} Title={'Delete ' + (record.AltValue ?? record.Value)}
                 CallBack={(conf) => { if (conf) Delete(); setShowWarning(false); }} />
-            <Modal Title={record.ID == 0 ? 'Add New Value List Item' : 'Edit ' + (record?.AltValue ?? record.Value)} Show={showModal} ShowCancel={false} ConfirmText={record.ID == 0 ? 'Add' : 'Save'}
+            <Modal Title={record.ID == 0 ? 'Add New Value List Item' : 'Edit ' + (record.AltValue ?? record.Value)} Show={showModal} ShowCancel={false} ConfirmText={record.ID == 0 ? 'Add' : 'Save'}
```

Comment:
This may cause issues without the `?`. Did you verify `record` can not be null`? I assume `record` is `null` if no record is selected, and I think it will still render the Title (not sure about the children)

Reply:
NVM I just realized if `record` is `null` it will already fail in `record.ID`